bluemira/codes/process/mapping.py

```diff
@@ -26,27 +26,37 @@
 
 IN_mappings = {
     "P_el_net": ("pnetelin", "MW"),
+    "tau_flattop": ("tbrnmn", "s"),
     "n_TF": ("n_tf", "dimensionless"),
     "TF_ripple_limit": ("ripmax", "%"),
     "C_Ejima": ("gamma", "dimensionless"),
     "e_nbi": ("enbeam", "keV"),
     "P_hcd_ss": ("pinjalw", "MW"),
     "eta_nb": ("etanbi", "dimensionless"),
     "e_mult": ("emult", "dimensionless"),
-    "tk_sh_out": ("shldoth", "m"),
-    "tk_sh_top": ("shldtth", "m"),
-    "tk_sh_bot": ("shldlth", "m"),
-    "tk_vv_out": ("d_vv_out", "m"),
-    "tk_vv_top": ("d_vv_top", "m"),
-    "tk_vv_bot": ("d_vv_bot", "m"),
     "tk_cr_vv": ("ddwex", "m"),
     "tk_tf_front_ib": ("casthi", "m"),
     "tk_tf_side": ("casths", "m"),
     "PsepB_qAR_max": ("psepbqarmax", "MW.T/m"),
+    "q_0": ("q0", "dimensionless"),
+    "m_s_limit": ("m_s_limit", "dimensionless"),
+    "delta": ("triang", "dimensionless"),
+    "sigma_tf_case_max": ("sig_tf_case_max", "Pa"),
+    "sigma_tf_wp_max": ("sig_tf_wp_max", "Pa"),
+    "sigma_cs_wp_max": ("alstroh", "Pa"),
+    "H_star": ("hfact", "dimensionless"),
+    "bb_pump_eta_el": ("etahtp", "dimensionless"),
+    "bb_pump_eta_isen": ("etaiso", "dimensionless"),
+    "bb_t_inlet": ("inlet_temp", "K"),
+    "bb_t_outlet": ("outlet_temp", "K"),
+    "eta_ecrh": ("etaech", "dimensionless"),
+    "gamma_ecrh": ("gamma_ecrh", "1e20 A/W/m^2"),
```

Comment:
Is this just `Am^2/W` or do we need some brackets (this is what pint reads it as)?